.travis.yml

```
@@ -40,6 +41,32 @@ stages:
 
 jobs:
   include:
+  # opensuse: "recent" systems with ICB
+  #   note: when you PR, docker-cp provides, in the container, the branch associated with the PR (not master where there's nothing new)
+  #         1. docker create --name mobydick IMAGE CMD        <=> create a container (= instance of image) but container is NOT yet started
+  #         2. docker cp -a ${TRAVIS_BUILD_DIR} mobydick:/tmp <=> copy git repository (CI worker, checkout-ed on PR branch) into the container
+  #                                                               note: docker-cp works only if copy from/to containers (not images)
+  #         3. docker start -a mobydick                       <=> start to run the container (initialized with docker-cp)
+  - stage: opensuse
+    dist: bionic
+    script: |
+      sudo docker pull opensuse/tumbleweed                                   \
+      &&                                                                     \
+      sudo docker create --name mobydick opensuse/tumbleweed /bin/bash -c    \
+      "zypper install -y git gcc gcc-fortran gcc-c++ openmpi-devel        && \
```

```suggestion
      "zypper install -y git gcc gcc-fortran gcc-c++ openmpi2-devel        && \
```

Best to be specific here: openmpi-devel does pull in openmpi2-devel _now_ but it may change later.